runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -613,6 +617,34 @@ fn contract_compile_base_per_byte_v2(ctx: &mut EstimatorContext) -> (GasCost, Ga
     ctx.cached.compile_cost_base_per_byte_v2 = Some(costs.clone());
     costs
 }
+fn pure_deploy_bytes(ctx: &mut EstimatorContext) -> GasCost {
+    // 10kiB
+    let small_code = read_resource(if cfg!(feature = "nightly_protocol_features") {
+        "test-contract/res/nightly_small_contract.wasm"
+    } else {
+        "test-contract/res/stable_small_contract.wasm"
+    });
+    // 1MiB
+    let large_code = read_resource(if cfg!(feature = "nightly_protocol_features") {
+        "test-contract/res/nightly_large_contract.wasm"
+    } else {
+        "test-contract/res/stable_large_contract.wasm"
+    });
```

Comment:
I think for this cost it doesn't really matter which one we use, so we can always pick `stable_large_contract.wasm`?

Alternatively, perhaps it'd be even better to synthesize empty wasm "by hand" here.

Reply:
Ahh yeah this is indeed unnecessary to differentiate nightly/stable.

But what is the benefit of using a synthesized wasm?

Reply:
>But what is the benefit of using a synthesized wasm?

It'd be simpler -- `stable_large_contract.wsm` is a complex beast with all kinds of stuff inside. While it seems that what we want here is `(module (data "bla-bla-bla-bla-bla..."))`